src/services/auth/auth.service.ts

```diff
@@ -44,15 +46,17 @@ export class AuthService {
   }
 
  public isAuthenticated(): Observable<boolean> {
-    return this.envService.getEnv$().map( config => {
-      const jwt = this.cookieService.get(config.cookies.token);
-      if (!jwt) {
-        return false;
-      }
-      const jwtData = this.decodeJwt(jwt);
-      // do stuff!!
-      return jwtData.exp > Math.round(new Date().getTime() / 1000);
-    });
 
+    return this.httpService.get<boolean>('/auth/isAuthenticated');
+    // return this.envService.getEnv$().map( config => {
+    //   debugger
+    //   const jwt = this.cookieService.get(config.cookies.token);
+    //   if (!jwt) {
+    //     return false;
+    //   }
+    //   const jwtData = this.decodeJwt(jwt);
+    //   // do stuff!!
+    //   return jwtData.exp > Math.round(new Date().getTime() / 1000);
+    // });

```

Comment:
Do you want to keep this here?

Reply:
@adnanakgun removed the commented code. Thanks.